app/assets/javascripts/discourse/app/controllers/users.js

```diff
@@ -57,4 +64,14 @@ export default Controller.extend({
   loadMore() {
     this.model.loadMore();
   },
+
+  @action
+  updateGroupParam(selectedGroupId, currentSelection) {
+    this.set(
+      "group",
+      currentSelection.length
+        ? currentSelection[currentSelection.length - 1].name
```

Comment:
The reason for doing this here is, in the select kit, the expected behavior is when we use `singleSelect=true` property the user can select an option. but to change that selected option user first needs to remove the selected option first and add a new selection.

So, to make it simple to use this logic handles the case easily i.e. select one option, to change selected option you can directly choose other option. no need to remove first selection.

thoughts?